src/verify-valid-url.js

```diff
@@ -23,4 +22,21 @@ exports.verifyValidUrl = async function (req, url) {
       );
     }
   }
+}
+
+exports.verifyValidUrl = verifyValidUrl;
+
+exports.findUrlsToValidateInScopes = function (scopes) {
+  const toValidateUrls = [];
+  Object.keys(scopes).map((scope) => {
+    const scopeOverrides = Object.entries(scopes[scope]);
+    scopeOverrides.map(([specifier, address]) => {
+      // we cannot validate relative urls
+      if (scope.includes("http")) {
+        toValidateUrls.push(new URL(address, scope).toString());
```

Comment:
Relying on `toString()` in javascript is always a bit scary - is better to use the `href` property on URLs:

https://developer.mozilla.org/en-US/docs/Web/API/URL/href

```suggestion
        toValidateUrls.push(new URL(address, scope).href);
```